tetrateci/tsb-util/tsb_util.py

```diff
@@ -0,0 +1,272 @@
+import urllib.request
+import os, sys, shutil, platform
+import argparse
+import config
+import certs
+from subprocess import PIPE, Popen
+from jinja2 import Template
+
+# https://stackoverflow.com/a/23796709
+def cmdline(command):
+    process = Popen(args=command, stdout=PIPE, shell=True)
+    return process.communicate()[0]
+
+def print_cmdline(command):
+    print(str(cmdline(command), "utf-8"), end="")
+
+def apply_from_stdin(ns, yaml):
+    cmd = "cat << EOF | kubectl apply " + " -n " + ns + " -f - \n" + yaml + "\nEOF\n"
+    print_cmdline(cmd)
+
+cleanup_script = ""
+
+def create_namespace(index):
+    global cleanup_script
+    ns = "bookinfo" + str(index)
+    print("Create Namespace : " + ns)
+    print_cmdline("kubectl create ns " + ns)
+    print_cmdline("kubectl label namespace " + ns + " istio-injection=enabled")
+    cleanup_script += "kubectl delete ns " + ns + "\n"
+    return ns
+
+def install_bookinfo(conf, default_context):
+    global cleanup_script
+
+    # tenants
+    print_cmdline("kubectl create ns tetrate")  # need an for multiple
```

Comment:
ns?

Reply:
Should I create a new namespace for the tsb objects? If not where should they be residing?